src/index.html

```diff
@@ -8,9 +8,60 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style_bar.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="#"
+      method="get"
+      class="search_big"
+      data-qa="big"
+    >
+
+      <button type="submit" class="but_search_big">
+        <img
+          class="image_big"
+          src="./images/Search.svg"
+          alt="img_button"
+        >
+      </button>
+
+      <input
+        class="window_for_big"
+        type="text"
+        name="search"
+        required
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+
+    </form>
+
+    <form
+      action="#"
+      method="get"
+      class="search_small"
+      data-qa="small"
+    >
+
+      <button type="submit" class="but_search_small">
+        <img
+        class="image_small"
+        src=" ./images/Search.svg"
+        alt="img_button"
+      >
+      </button>
+
+      <input
+        class="window_for_small"
+        type="text"
+        name="search"
+        required
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+
+    </form>
```

Comment:
Let's add empty line only between logical blocks

```suggestion
    <form
      action="#"
      method="get"
      class="search_small"
      data-qa="small"
    >
      <button type="submit" class="but_search_small">
        <img
        class="image_small"
        src=" ./images/Search.svg"
        alt="img_button"
      >
      </button>

      <input
        class="window_for_small"
        type="text"
        name="search"
        required
        placeholder="Try &quot;Los Angeles&quot;"
      >
    </form>
```